gitops/prer/create_gitops_prs.go

```diff
@@ -100,6 +103,40 @@ func bazelQuery(query string) *analysis.CqueryResult {
 	return qr
 }
 
+func getContext(workdir *git.Repo, branchName string) map[string]interface{} {
+	commitSha := workdir.GetCommitSha()
+
+	utcDate, err := exec.Ex("", "date", "-u")
+	if err != nil {
+		log.Fatal(err)
+	}
+	utcDate = strings.TrimSpace(utcDate)
+
+	ctx := map[string]interface{}{
+		"GIT_REVISION": commitSha,
+		"UTC_DATE":     utcDate,
+		"GIT_BRANCH":   branchName,
+	}
+
+	return ctx
+}
+
+func stampFile(fullPath string, workdir *git.Repo, branchName string) {
+	template, err := os.ReadFile(fullPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	ctx := getContext(workdir, branchName)
+
+	stampedTemplate := fasttemplate.ExecuteString(string(template), "{{", "}}", ctx)
```

Comment:
`fasttemplate.Execute` function writes directly into file.
```suggestion
        outf, err = os.OpenFile(fullPath, os.O_RDWR|os.O_CREATE|os.O_TRUNC, perm)
	if err != nil {
		log.Fatalf("Unable to create output file %s: %v", output, err)
	}
	defer outf.Close()
	_, err = fasttemplate.Execute(string(template), "{{", "}}", ctx)
```